runtime/isp_run_app.py

```diff
@@ -246,7 +247,7 @@ def main():
                 sys.exit(1)
 
         if not os.path.isfile(pex_path):
-            if compileMissingPex(policy_dir, pex_path, args.simulator, args.arch, args.extra) is False:
+            if compileMissingPex(policy_dir, pex_path, args.simulator, arch, args.extra) is False:
```

Comment:
This looks like it fixes architecture autodetect. Is that true?

Reply:
I think it's the last one of these still hanging out.